utils/node_local_test.py

```diff
@@ -2561,13 +2630,16 @@ def main():
             fatal_errors.add_result(run_posix_tests(server, conf))
         else:
             fatal_errors.add_result(run_posix_tests(server, conf, args.test))
-    else:
+    elif not args.mode:
         fatal_errors.add_result(run_il_test(server, conf))
         fatal_errors.add_result(run_dfuse(server, conf))
         fatal_errors.add_result(run_posix_tests(server, conf))
         fatal_errors.add_result(set_server_fi(server))
+    else:
+        print("Unknown mode")
+        fatal_errors.add_result(True)
 
-    if server.stop(wf_server) != 0:
+    if server.stop(wf_server, abort_on_warning = server_log_strict) != 0:
```

Comment:
(pylint-redundant-keyword-arg) Argument 'abort_on_warning' passed by position and keyword in method call